api/src/client/WebserviceLicenseStatusProcessorClient.test.ts

```diff
@@ -22,14 +23,15 @@ const entityWithAddress = (address: string): LicenseEntity => {
 describe("WebserviceLicenseStatusProcessorClient", () => {
```

Comment:
Boulder - Since `updateLicenseStatusFactory`, `userRouterFactory`, and `licenseStatusRouterFactory` use this function, I'd expect each to have a test that validates the error from the line `throw new Error(Cannot find license status ${value})` is thrown.